### PR TITLE
Temp disable of inspections

### PR DESCRIPTION
**Pull request by romani**
Issue #11566

### CODE REVIEW

File: config/intellij-idea-inspections.xml [hunk 1]
```diff
@@ -3768,6 +3774,9 @@
     <option name="ignoreEmptySuperMethods" value="false"/>
     <option name="onlyReportWhenAnnotated" value="true"/>
   </inspection_tool>
+  <!-- temporal suppression until issues are resolved -->
```
Comment (rnveach) [inline config/intellij-idea-inspections.xml:3777]: Can you tie this to an issue number if this is going to be merged as is?

Comment (romani) [reply config/intellij-idea-inspections.xml:3777]: I need to create separate issue, as #11566 is already overused for numerous updates

Comment (romani) [reply config/intellij-idea-inspections.xml]: done

**Review by rnveach — APPROVED** [review]

**Comment by romani** [comment]
@pbludov , fyi. Let me know if concerns on some updates.

Grammar violations, I do not know how to suppress, I will play more next day

**Pull request merged**